website/models.py

```diff
@@ -805,6 +805,13 @@ def __str__(self):
         return f"{self.user.username} - {self.amount} BACON"
 
 
+class Trademark(models.Model):
+    name = models.TextField(max_length=200, null=True, blank=True)
+    domain = models.ForeignKey(Domain, null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
you mean Company or Domain model name ?

Reply:
I think it would be better to add Trademark model to Domain than company since in edit domain It would be easier to manage and view trademark and also trademarks can be associated with the domains where as same company will have different domain and different trademark.
should I Procced ?

Reply:
Done